examples/framework/preferences/minimalEditor/README.md

```diff
@@ -0,0 +1,9 @@
+
+## Minimal Preferences Editor ##
+
+This folder provides an example of a minimal but fully functional preferences editor. It has:
+- only one preference
+- no enactor
+- no localization (Note that this is not best practice!)
+
+This code is the basis for the Preferences Framework tutorial.
```

Comment:
No, we haven't yet identified an URL for it, but you're right: We should link to it.